### ISSUE TITLE
How to fine tune pretrained  ResNet model in slim with tensorlayer?

### ISSUE THREAD

**Issue by donproc** [zsdonghao/tensorlayer#118]
I can set network by tl.layers.SlimNetsLayer and can also restore model. My question is  how to set logits to 'restore=False' without modifying the code in resnet_v1.py. Or is there a way to modify?
Thanks for your help.

**Comment by donproc** [comment]
Solved

**Comment by baiyancheng20** [comment]
@donproc I also met with this problem. Could you give me a detailed discription on how to finetuning?

**Comment by li-zemin** [comment]
@baiyancheng20 Hello, did you solve the problem, I want to extract feature of image from resnet, do you know the methods?
